Form/AddBaseUserType.php

```diff
@@ -38,6 +51,21 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             [
                 'type' => PasswordType::class,
             ]
+        )->add(
+            'roles',
+            EntityType::class,
+            [
+                'attr' => [
+                    'class' => 'select2',
```

Comment:
This seems oddly specific. I'd remove this.

Reply:
We implement the select2 entity bundle in the multi user bundle to have the correct functionality of the user role selection. The other option is to implement it per project but that means we have to set it up each time, so for now we will just implement it in the bundle.